modules/tensor_mechanics/src/actions/TensorMechanicsAxisymmetricRZAction.C

```diff
@@ -6,74 +6,14 @@
-  params.addParam<std::vector<AuxVariableName> >("save_in", "The displacement residuals");
-  params.addParam<std::vector<AuxVariableName> >("diag_save_in", "The displacement diagonal preconditioner terms");
-  return params;
+  return validParams<TensorMechanicsAction>();
 }
 
 TensorMechanicsAxisymmetricRZAction::TensorMechanicsAxisymmetricRZAction(const InputParameters & params) :
-    Action(params)
-{
-}
-
-void
-TensorMechanicsAxisymmetricRZAction::act()
+    TensorMechanicsAction(params)
 {
-  std::vector<NonlinearVariableName>displacements = getParam<std::vector<NonlinearVariableName> >("displacements");
-  unsigned int _ndisp = displacements.size();
-
-  //Error checking:  Can only take two displacement variables in StressDivergenceRZTensors kernel
-  if (_ndisp != 2)
-    mooseError("Number of displacement variables should be 2 but recieved " << _ndisp);
-
-  std::vector<VariableName>coupled_displacements;
-  for (unsigned int i = 0; i < _ndisp; ++i)
-    coupled_displacements.push_back(displacements[i]);
-
-  std::vector<AuxVariableName>save_in = getParam<std::vector<AuxVariableName> >("save_in");
-  if (isParamValid("save_in") && save_in.size() != _ndisp)
-    mooseError("Number of save_in variables should equal to the number of displacement variables " << _ndisp);
-
-  std::vector<AuxVariableName>diag_save_in = getParam<std::vector<AuxVariableName> >("diag_save_in");
-  if (isParamValid("diag_save_in") && diag_save_in.size() != _ndisp)
-    mooseError("Number of diag_save_in variables should equal to the number of displacement variables " << _ndisp);
-
-  InputParameters params = _factory.getValidParams("StressDivergenceRZTensors");
-  params.set<std::vector<VariableName> >("displacements") = coupled_displacements;
-
-  params.set<bool>("use_displaced_mesh") = getParam<bool>("use_displaced_mesh");
-
-  if (isParamValid("base_name"))
-    params.set<std::string>("base_name") = getParam<std::string>("base_name");
-
-// Check whether this StressDivergenceRZTensors kernel is restricted to certain block?
-  if (isParamValid("block"))
-    params.set<std::vector<SubdomainName> >("block") = getParam<std::vector<SubdomainName> >("block");
-
-  for (unsigned int i = 0; i < _ndisp; ++i)
-  {
-    std::string kernel_name = "TensorMechanicsAxisymmetricRZ_" + Moose::stringify(i);
-
-    params.set<unsigned int>("component") = i;
-    params.set<NonlinearVariableName>("variable") = displacements[i];
-  if (isParamValid("save_in"))
-    params.set<std::vector<AuxVariableName> >("save_in") = {save_in[i]};
-  if (isParamValid("diag_save_in"))
-    params.set<std::vector<AuxVariableName> >("diag_save_in") = {diag_save_in[i]};
-
-    _problem->addKernel("StressDivergenceRZTensors", kernel_name, params);
-  }
+  mooseDeprecated("Use the 'TensorMechanics' action instead. It autodetects the coordiante system.");
```

Comment:
Ha, good catch. Grepping for that typo I discovered it is also here:
https://github.com/libMesh/libmesh/blob/master/contrib/metis/GKlib/pdb.c#L442